ArjunaJTS/jts/pom.xml

```diff
@@ -192,6 +192,24 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
@jmfinelli please can you point out where this is used in the build after this change?

Reply:
The test jar gets extracted [here](https://github.com/jbosstm/narayana/pull/2049/files#diff-d4f42289a362c7d991ef7305ebf659ea59d80c9fcb0e3ee42fc80f79f457aa92R100)

Reply:
Although I think your link is not going to a location that I can find, I do see the use of `>jts<` for `<classifier>test-sources</classifier>` in qa/pom.xml but I can't see `<classifier>sources</classifier>` - is it definitely needed here?

Reply:
Basically, I tried to recreate the behaviour I noticed before adding `<classifier>sources</classifier>`. In other words, the source jar was already generated and put in the `target` folder, that is why I added the `<classifier>sources</classifier>` to the plugin.